.gitignore

```diff
@@ -7,8 +7,6 @@ lib/core/MetadataBlog.js
 
 website/translated_docs
 website/build/
-website/yarn.lock
```

Comment:
Do we want to add `package-lock` to gitignore in order to keep the "no lock file approach" or are we planning to fix it with this PR?

Reply:
I think we want to allow the lockfile for the site to alleviate build indeterminacy.